src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -967,6 +967,17 @@ class MacroAssembler: public Assembler {
                                           Register g, Register h, int iteration);
 
   void addmq(int disp, Register r1, Register r2);
+
```

Comment:
All Poly1305-related methods can be moved to `StubGenerator`. They are used solely during stub creation.